src/main/java/no/nav/familie/integrasjoner/client/rest/OppgaveRestClient.kt

```diff
@@ -41,7 +41,7 @@ class OppgaveRestClient(@Value("\${OPPGAVE_URL}") private val oppgaveBaseUrl: UR
 
     private val logger = LoggerFactory.getLogger(OppgaveRestClient::class.java)
 
-    fun finnOppgave(request: Oppgave): Oppgave {
+    fun finnÅpenBehandleSakOppgave(request: Oppgave): Oppgave {
```

Comment:
er det ikke riktigere å bruke finnÅpenOppgave som metodenavn, i og med at oppgavetype er input?

Reply:
Nei. For oppgavetype er hardkodet til BEH_SAK i request